sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -79,20 +96,3 @@ Note that these breaking changes are only breaking changes from the **preview**
 - [Azure Digital Twins Public Repo](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core)
 - [Azure Digital Twins Samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples)
 
-## 1.0.0-preview.1 (Unreleased)
```

Comment:
This looks like a case where the date never got stamped in.

Reply:
It was never released so I removed it.